config/aso/kustomization.yaml

```diff
@@ -17,23 +17,14 @@ patches:
       - op: test
```

Comment:
nit: the comment on the line above is no longer entirely accurate, so I'd vote to remove it.

Reply:
Ack